src/BrewingRecipes.cpp

```diff
@@ -45,8 +45,9 @@ void cBrewingRecipes::ReloadRecipes(void)
 			ParsingLine.erase(FirstCommentSymbol);
 		}
 
-		if (ParsingLine.empty())
+		if (std::all_of(ParsingLine.cbegin(), ParsingLine.cend(), isspace))
```

Comment:
It seems it could be useful to extract this test into a separate function, `IsOnlyWhitespace`, somewhere in StringUtils. How about that?